.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,64 @@
+---
+name: k6 Release 
+about: k6 release accommodates activities and a checklist that has the k6 open-source release process.
+title: 'k6 release v0.4x.0'
+---
+
+**Release Date**: 
+
+25th May 2023 <<  TODO: WRITE HERE THE UPDATED RELEASE DATE
+
+## Release Activities
+
+### At the beginning of the cycle
+
+- [ ] Create a new branch `release-notes-v04x` and add the new related release notes file.
+      
+All team members are expected to open PRs for adding new release notes items when they add a new change to the k6 code base. The issue's assignee should remember from time to time to do it, if some PRs are forgotten.
+```
+release-notes-v04x <--- my-single-feature-notes-branch
```

Comment:
Sorry, I didn't consider well about this in the previous PR.

I prefer not to have it here, but create an internal document (e.g., "What is k6's cycle" ) to mention this and provide all details.

The rationale is to avoid turning this issue into documentation since nobody usually expects to use the issue as _process_ documentation.

```suggestion
```

Reply:
And hardly against responsibilities like `from time to time to push somebody.` First, I'd prefer a process that helps grow responsibilities. Second, not sure if we need that action more than once in a cycle to avoid spending the release driver's time on that chore.

Reply:
>I prefer not to have it here, but create an internal document

I'm not against it but we said we are going to have these documents public so I think we need to create this doc as part of the docs folder. Can you take care of it? Then we can link it from here and from the internal docs.

Reply:
I think we were discussing to making the public the checklist, not our internal docs.

>Can you take care of it? 

I was under the impression that you're driving this, but okay for sure a can take the lead.

Reply:
I'm driving the template scope, I appreciate if you could help me doing in parallel this new doc.

Reply:
I removed the documentation